modules/light-clients/07-tendermint/expected_keepers.go

```diff
@@ -0,0 +1,16 @@
+package tendermint
```

Comment:
we can optionally move `expected_keepers.go`, `migrations.go`, and `migrations_test.go` to its own package. 

Then the import would be:
```go
ibctmmigrations.PruneTendermintConsensusStates
```

Maybe we can rename the function to `PruneExpiredConsensusStates`?

Reply:
I like the idea of moving these to their own package. I do agree, I'm not a huge fan of `ibctmmigrations` but it would just be another import alias and it could be aliased however a chain dev sees fit. 
I think the trade off for us is better because we can maintain the migration code and expected keepers file separately, at the expense of a kind of ugly import alias which we don't really have to worry about on a day to day.

I'd be leaning towards the migrations pkg personally.

Reply:
If we don't expect these functions to be used outside the context of migrations I think a migrations package makes sense. Also not a huge fan of import alias but it's not a huge deal breaker.

Reply:
I think pruning which would be performed outside of migrations would need to be optimized (not just iterating over all clients, but maybe a specific set of clients), so I think it makes sense to add this to a separate package and implement the optimized approach (which could maybe be executed via a governance proposal) at some other point, especially since `PruneAllExpiredConsensusStates` will continue to be a funciton of 07-tendermint

Reply:
updated